cmd/tiller-proxy/internal/handler/apprepos_handler.go

```diff
@@ -137,6 +142,7 @@ func (a *appRepositoriesHandler) Create(w http.ResponseWriter, req *http.Request
 	if a.kubeappsNamespace == "" {
```

Comment:
just realized this too, you should be able to use the env var instead of storing the namespace in the struct.